src/main/kotlin/ink/abb/pogo/scraper/Settings.kt

```diff
@@ -93,6 +94,17 @@ class Settings(val properties: Properties) {
         listOf()
     }
 
+    val autoEvolve = getPropertyIfSet("list of pokemon you want to evolve when able to", "auto_evolve", "CATERPIE,PIDGEY,WEEDLE", String::toString).split(",")
+
+    //This method only exists because I wasn't sure if this data was stored somewhere else. If it is then this can be removed.
+    private fun getCandyByPokemon(): Map<Int, Int> {
+            val lines = File("pokemon-candy.csv").readLines()
```

Comment:
"IF PokeBank > # (or FULL)
USE Lucky Egg
EVOLVE for 30 Mins"

as far as lucky egg I think it would be better to have bot count number of possible evolutions from listing in config aka


+# list of pokemon you want to auto evolve
 +auto_evolve=PIDGEY,weedle,caterpie
+# Minimum Number of evolutions possible before Auto, Leave at -1 or blank to evolve always.
 +auto_evolvemin=50
+# Use Lucky Egg before Evolve cycle
 +auto_evolveUseEgg=true

so that would wait for 50 possible evolutions from that list, then use lucky egg.

+# list of pokemon you want to auto evolve
 +auto_evolve=PIDGEY,weedle,caterpie
+# Minimum Number of evolutions possible before Auto, Leave at -1 or blank to evolve always.
 +auto_evolvemin=-1
+# Use Lucky Egg before Evolve cycle 
 +auto_evolveUseEgg=false

Would evolve every pidgey etc after a catch. no egg